specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,73 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for humans and systems. The FOCUS specification does not require a specific level of precision for numeric values. The level of precision required for a given column is determined by the provider and should be part of a data definition published by the provider.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST follow the formatting requirements listed below. Provider generated numeric value capturing columns SHOULD adopt the same format requirements over time.
```

Comment:
>  Provider generated numeric value capturing columns SHOULD adopt the same format requirements over time.

I would propose a less strong statement:

>  Provider generated numeric value capturing columns SHOULD adopt *backward-compatible* format requirements over time.

Reply:
I implemented this suggestion.  Thanks